edb/server/compiler/compiler.py

```diff
@@ -812,6 +817,41 @@ def compute_stmt_name(text: str) -> str:
 
         return sql_units
 
+    def compile_request(
+        self,
+        user_schema: s_schema.Schema,
```

Comment:
These should be kwargs

Reply:
Tracked in https://github.com/edgedb/edgedb/issues/6879